ViewModel/ExpressCheckoutButton.php

```diff
@@ -84,7 +84,11 @@ public function shouldRender(): bool
      */
     public function isExpressCheckoutActive(): bool
```

Comment:
why is the method called `isExpressCheckoutActive` when it is really semantically `isExpressCheckoutInactive`??

Reply:
this old method, we can change the name if not reflected.
@Skullsneeze ?